controllers/accounts.js

```diff
@@ -1,104 +1,68 @@
 const accountModel = require('../models/accounts')
 
 function signup(req, res, next) {
-
-    const {
-        petname,
-        username,
-        password
-        //   displayname,
-        //   profilepic,
-        //   eatinghabits,
-        //   quirks,
-        //   bio,
-        //   age,
-        //   type,
-        //   interactions
-    } = req.body
-    // if(!displayname || !profilepic || !eatinghabits || !quirks || !bio || !age || !type)
-    if (!petname || !username || !password) return next({
-        status: 400,
-        message: 'Incomplete signup'
+  const { username, password, displayname, profilepic, eatinghabits, quirks, bio, age, type, interactions } = req.body
+  if (!username && !password)
+    return next({
+      status: 400,
+      message: 'Username and Password required for creating an account'
     })
-
```

Comment:
the `petname` was not updated in the seed/migration files. front-end also uses `displayname`. will remove the other stuff though